cupy_backends/cuda/libs/nvtx.pyx

```diff
@@ -144,9 +144,22 @@ cpdef RangePushC(message, uint32_t color=0):
     """
     Starts a nested range.
 
-    Ranges are used to describe events over a time span during execution of
-    the application. The duration of a range is defined by the corresponding
-    pair of ``RangePush*()`` to ``RangePop()`` calls.
+    Ranges are used to describe events over a time span during execution of the
+    application. This is particularly useful when profiling with Nsight Systems
+    to help connect user-specified ranges with CuPy's internal CUDA-kernels.
+    The duration of a range is defined by the corresponding pair of
+    ``RangePushC()`` to ``RangePop()`` calls, which can be nested.
+
+    Example:
```

Comment:
I could be wrong, but I guess Sphinx expects the `Examples` section to have the Python prompts. How about trying this instead?
```suggestion
    .. code-block:: python
```